tests/Serval.SpecFlowTests/StepDefinitions/MachineApiStepDefinitions.cs

```diff
@@ -0,0 +1,337 @@
+using System.Net;
+using NUnit.Framework;
+
+namespace Serval.SpecFlowTests.StepDefinitions;
+
+[Binding]
+public sealed class MachineApiStepDefinitions
+{
+    // QA int server: "https://machine-api.org/"
+    // QA ext server: "https://qa.serval-api.org/"
+    // localhost: "http://localhost/"
+    const string MACHINE_API_TEST_URL = "http://localhost/";
+    readonly Dictionary<string, string> EnginePerUser = new();
+    readonly Dictionary<string, string> CorporaPerName = new();
+
+    // For additional details on SpecFlow step definitions see https://go.specflow.org/doc-stepdef
+    private readonly HttpClient httpClient;
+    private readonly DataFilesClient dataFilesClient;
+    private readonly TranslationEnginesClient translationEnginesClient;
+    private string? bearer = null;
+
+    private TranslationBuildConfig translationBuildConfig = new TranslationBuildConfig
+    {
+        Pretranslate = new List<PretranslateCorpusConfig>()
+    };
+
+    public MachineApiStepDefinitions()
+    {
+        //ignore ssl errors
+        var handler = new HttpClientHandler();
+        handler.ClientCertificateOptions = ClientCertificateOption.Manual;
+        handler.ServerCertificateCustomValidationCallback = (httpRequestMessage, cert, cetChain, policyErrors) =>
+        {
+            return true;
+        };
+        httpClient = new HttpClient(handler);
+        httpClient.BaseAddress = new Uri(MACHINE_API_TEST_URL);
+
+        dataFilesClient = new DataFilesClient(httpClient);
+        translationEnginesClient = new TranslationEnginesClient(httpClient);
+    }
+
+    private async void setBearer()
+    {
+        if (bearer is not null)
+        {
+            return;
+        }
+        bearer = await Utilities.GetAccessTokenFromEnvironment(httpClient);
+        httpClient.DefaultRequestHeaders.Add("authorization", $"Bearer {bearer}");
+    }
+
+    [Given(@"a new (.*) engine for (.*) from (.*) to (.*)")]
+    public async Task GivenNewEngine(
+        string engineTypeString,
+        string user,
+        string source_language,
+        string target_language
+    )
+    {
+        setBearer();
+        var existingTranslationEngines = await translationEnginesClient.GetAllAsync();
```

Comment:
You should only use `var` when the type is explicit elsewhere in the line. This helps to make the code more readable.

Reply:
Updated.  Will check in the future for these types of things for clarity.